libraries/AP_Logger/AP_Logger_Backend.cpp

```diff
@@ -675,7 +675,7 @@ void AP_Logger_Backend::vehicle_was_disarmed()
     }
 }
 
-// this sensor is enabled if we should be logging at the moment
+// we report enabled if we should be logging at the moment
```

Comment:
I think this is a backwards step, assuming we use this more than "reporting" to the GCS.